src/sdk/sql_cluster_router.cc

```diff
@@ -4130,6 +4146,7 @@ std::shared_ptr<hybridse::sdk::ResultSet> SQLClusterRouter::ExecuteShowTableStat
                     partition_unalive++;
                 }
             }
+            CheckTableStatus(db, table_name, tid, partition_info, replica_num, &error_msg);
```

Comment:
it will send  many `getablestatus` requests and cost much time if there are many tables. It's better to get all status info in one `gettablestatus` request